components/raftstore/src/store/peer.rs

```diff
@@ -670,8 +670,10 @@ where
     pub compaction_declined_bytes: u64,
     /// Approximate size of the region.
     pub approximate_size: Option<u64>,
+    pub may_split_size: Option<bool>,
```

Comment:
FYI, maybe we can combine the `may_split_size` and `may_split_keys` into one object, naming as `maybe_splittable` to represent the given `Peer` maybe splittable by size or keys?
```Rust
/// Mark this region maybe splittable in previous check.
pub maybe_splittable: Option<bool>,
```

Reply:
There's a problem with single maybe_splittable. Because what if keys checker says it's not splitable, but size check says it's splitable? The maybe_splittable could be incorrectly overwritten by either of the value (though today the caller use same splitable value, it's possible in the future it can be different). So having two variables is more robust.